src/libs/actions/Report.js

```diff
@@ -800,21 +812,18 @@ function addAction(reportID, text, file) {
  * network layer handle the delayed write.
  *
  * @param {Number} reportID
- * @param {Number} sequenceNumber
+ * @param {Number} [sequenceNumber]
  */
 function updateLastReadActionID(reportID, sequenceNumber) {
-    const currentMaxSequenceNumber = reportMaxSequenceNumbers[reportID];
-    if (sequenceNumber < currentMaxSequenceNumber) {
-        return;
-    }
+    const lastReadSequenceNumber = sequenceNumber || reportMaxSequenceNumbersNotLoading[reportID];
 
-    setLocalLastRead(reportID, sequenceNumber);
+    setLocalLastRead(reportID, lastReadSequenceNumber, sequenceNumber !== undefined);
```

Comment:
e.g. based on my previous comments this would end up looking like this instead:

```js
setLocalLastRead(reportID, lastReadSequenceNumber);

if (sequenceNumber) {
    setNewMarkerPosition(sequenceNumber);
}
```

Reply:
Answered on the other comments why I think it should remain as-is